src/app/home-page/widget/organization-box/organization-box.component.scss

```diff
@@ -25,3 +25,8 @@ img[alt='Org avatar'] {
   width: 6rem;
   line-height: 1.5rem;
 }
+
+// Overide mat-card padding to extend mat-divider to edges
+mat-divider {
```

Comment:
This is repeated 3 times I think. It might be better to put this in a shared style sheet and import them to each component

Reply:
Added a shared sheet to `app/shared/styles/dashboard-boxes.scss` and moved over a few other shared styles, thank you!